### PR TITLE
Added C# Logic/Math Operators

### LINKED ISSUE #299: Add C# Logic/Math Operators
# Description

Logic/Math Operators for C# aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/operators.json` to the right C# directory in `web/thesauruses/`
* Edit the operators.json file to accurately reflect the latest major version of C#
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then leave `"code":` blank.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by VaishnaviNandakumar**
## What GitHub issue does this PR apply to?

Resolves #299 

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?
Additional operators (conditional_and, conditional_and_assignment, conditional_or, conditional_or_assignment) have been included.

### CODE REVIEW

**Comment by VaishnaviNandakumar** [comment]
Hi @geekygirlsarah! The initial commit also included the Source folder which had to be removed. However during the second commit, the update for issue #333 was also included. Please let me know if I have to remove the changes and submit a separate PR or if it can stay the same. Thanks!

File: web/thesauruses/_meta/operators.json [hunk 1]
```diff
@@ -51,6 +51,10 @@
       "right_shift_assignment"
     ],
     "Conditional Operators": [
+      "conditional_and",
+      "conditional_and_assignment",
+      "conditional_or",
+      "conditional_or_assignment",
```
Comment (geekygirlsarah) [inline web/thesauruses/_meta/operators.json]: For now, I'm going to say roll back all changes to `_meta/operators.json` as they need to be thought out more in issue #333 first before they're applied here. 

Similarly, remove the newer concept IDs (like `conditional_and`) from the C# file below so I can merge your PR but then we can edit the operators a little later after #333 is done.

File: web/thesauruses/csharp/operators.json [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    },
+    "increment": {
+      "code": "++"
+    },
+    "decrement": {
+      "code": "--"
+    },
+    "exponential": {
+      "not-implemented": "true",
+      "code": ""
```
Comment (geekygirlsarah) [inline web/thesauruses/csharp/operators.json]: YOu can remove `code` lines from anything that's `not-implemented: true`

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
So I think if you make those changes, I'd merge this in. I realize it's a lot to undo, so sorry about that. (Though I do appreciate you thinking these parts through!)

**Comment by VaishnaviNandakumar** [comment]
No worries! I've made the required changes. Since `_meta/operators.json` was updated with a few more operators  (unary plus, unary minus, integer division and integer division assignment) I added those too in `csharp/operators.json`. Do let me know if that's okay. Thanks.

**Comment by geekygirlsarah** [comment]
Thanks for making the changes _*AND*_ adding the extra new operators in!

I'll merge this in then. I appreciate it!

**Pull request merged**